templates/typescript/base.json

```diff
@@ -2,7 +2,7 @@
   "$schema": "https://json.schemastore.org/tsconfig",
   "display": "Default",
   "compilerOptions": {
-    "lib": ["es2022"],
+    "lib": ["es2022", "dom"],
```

Comment:
why do we need this?

Reply:
yeah I forgot to send the screenshot, but portal has a bunch of window/dom-related elements in their SDK. I had to add this to build, other was "window not defined" related errors when trying to build.